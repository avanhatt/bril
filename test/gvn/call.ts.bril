test/ts/call.ts.bril