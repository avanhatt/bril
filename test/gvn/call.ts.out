void main() {
b1:
  v01: int = const 5;
  x1: int = id v01;
  call callPrint(x1);
  ret ;
}
void callPrint(x : int) {
b1:
  print x;
  ret ;
}
